feature-libs/organization/administration/components/shared/item.service.spec.ts

```diff
@@ -150,5 +150,29 @@ describe('ItemService', () => {
         expect(form.disabled).toBeFalsy();
       });
     });
+
+    describe('isInEditMode', () => {
+      let result;
```

Comment:
you should clear or reset the result in `beforeEach`, or move this `let` inside `it`